src/interfaces/alteration-interfaces/business-snapshot-interface.ts

```diff
@@ -0,0 +1,17 @@
+import {
+  BusinessContactIF,
+  BusinessInformationIF,
+  IncorporationAddressIf,
+  NameTranslationSnapshotIF,
+  OrgPersonIF,
+  ShareStructureIF
+} from '@/interfaces'
+
+export interface BusinessSnapshotIF {
+  business: BusinessInformationIF,
+  aliases: Array<NameTranslationSnapshotIF>,
+  IncorporationAddressIf,
+  directors: OrgPersonIF,
+  shareClasses: ShareStructureIF,
+  BusinessContactIF
```

Comment:
This looks over-optimized 😛 
The sub-object should have a name (that isn't the same as the IF type), please.

Reply:
Fixed in next commit with the unit tests 👍